src/components/projects/create-wp-form/create-wp-form.tsx

```diff
@@ -75,20 +98,83 @@ const CreateWPForm: React.FC = () => {
     }
   };
 
-  const handleSubmit = (e: any) => {
+  const stateBundleBuilder = (state: any, setter: any) => {
```

Comment:
Better types, also move to diff file

Reply:
Function is no longer needed after removing the `StateBundle` interface